library/styleguideComponents/ReactComponent/index.js

```diff
@@ -65,13 +66,37 @@ export default class ReactComponent extends Component {
                 <div className="rainbow-p-top_small rainbow-p-horizontal_x-large">
                     <RenderIf isTrue={activeTabName === 'examples'}>
                         <div className="rainbow-m-left_x-large rainbow-m-right_xx-large">
+                            <div className="react-rainbow-component_example-edit-link-container">
+                                <a
+                                    className="react-rainbow-component_example-edit-link"
+                                    href={`https://github.com/90milesbridge/react-rainbow/blob/master/src/components/${name}/Readme.md`}
+                                    target="_blank"
+                                    rel="noopener noreferrer">
+                                    <Button variant="base">
```

Comment:
A button can't be inside an anchor. Put a general class for links in the Application component that looks like the button and then apply the class to the anchor.